pkg/components/controllers.go

```diff
@@ -176,8 +180,10 @@ func startIngressController(ctx context.Context, cfg *config.Config, kubeconfigP
 		return err
 	}
 
-	extraParams := assets.RenderParams{
+	extraParams = assets.RenderParams{
```

Comment:
Can we define it with these 3 fields straightaway on line 162?

Reply:
Done